docs/triage-party/triage-party-deployment.go

```diff
@@ -18,16 +18,14 @@ package main
 
 import (
 	"fmt"
-
+	
```

Comment:
This file should not change. 
Also [docs/proposal/20200506-single-controller-multitenancy-flow.svg](https://github.com/kubernetes-sigs/cluster-api-provider-aws/pull/3563/files#diff-189069f4133909854878c049610da1cbc5dd62c8f94b830887e200b86aee8d74) changes above.

Reply:
I think some make target messed it up, I ll revert it.